INSTALL.rst

```diff
@@ -335,7 +335,9 @@ before you get started.
   # Activate it
   conda activate skimage-dev
   # Install main development and runtime dependencies
-  conda install -c conda-forge --file requirements.txt
+  conda install -c conda-forge --file requirements/default.txt
```

Comment:
With `conda 23.1.0` I get the following error for this line
```
InvalidVersionSpec: Invalid version '1.9.2;python_version<='3.9'': invalid character(s)
```
Can anyone else reproduce?

Reply:
I think similar issues came up in the past. It seems that the requirement spec that pip uses can not be parsed by conda. And because packages might be named differently on conda-forge and PyPI I don't see a way around manually curating it, e.g. like [SciPy's `environment.yml`](https://github.com/scipy/scipy/blob/main/environment.yml).

Reply:
Hi @lagru,

> Can anyone else reproduce?

I just checked, because I was using `conda 22.11.1` when I submitted this PR; I am now using `conda 23.1.0` and running `conda install -c conda-forge --file requirements/default.txt` doesn't raise any error for me (after following each and every step from the installation guide).

I'm curious where the `python_version<='3.9'` requirement would come from, since we rather have requirements such as `python_version>='3.8'` (?).

> I think similar issues came up in the past.

Right, because the syntax for specifying version or platform constraints differs between `pip` and `conda` (plus what you said about package names on PyPI vs conda-forge). The `--file` vs `-r` flags for passing files is only one obvious (and easy enough to handle) difference...